setup.py

```diff
@@ -164,6 +164,7 @@ def configuration(parent_package='',top_path=None):
                        quiet=True)
 
     config.add_subpackage('numpy')
+    config.add_data_files(('numpy', '*.txt'))
```

Comment:
Files should be listed explicitly if we decide to install them. In general, installing (rather than only bundling) text files isn't the right thing to do.

Reply:
This comes from SciPy and is needed for the recent license updates for wheels where the tests for the license are run on the installed module. What do you mean by "bundled"?

Reply:
I'd be happy to remove all but the license file. I can confirm that the current NumPy wheels contain no license data.